app.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+'use strict';
+
+const http      = require('http');
+const socketio  = require('socket.io');
+const Kasocki   = require('kasocki');
+const P         = require('bluebird');
```

Comment:
Cool, I like maintaining it :D  unless you decided to enforce it, I'll leave it as is.